lib/generators/folio/install_generator.rb

```diff
@@ -87,7 +87,6 @@ def copy_templates
           "db/migrate/20220214083648_rm_private_attachments_mime_type_column.rb",
           "lib/tasks/developer_tools.rake",
           "public/maintenance.html",
-          "test/factories.rb",
           "test/test_helper.rb",
```

Comment:
@foton Takhle tam ale ten soubor vubec nevznikne. Soubor tam chceme.

Reply:
Ano, to je to kopírování,, které se snažím nahradit linkováním.

Reply:
Takhle to ale zustalo v templates. Radeji nez vyhazovat odsud bych upravil ten soubor. Abych ho nemusel v novem projektu zakladat.